frontend/public/locales/zh/machine-health-checks.json

```diff
@@ -0,0 +1,15 @@
+{
+  "Name": "名称",
+  "Namespace": "命名空间",
+  "Created": "创建的",
+  "MachineHealthChecks": "MachineHealthChecks",
+  "Unhealthy conditions": "不健康条件",
+  "Status": "状态",
+  "Timeout": "timeout",
```

Comment:
"timeout", same as other timeout.

Reply:
No change here.

Reply:
I can see that this had been addressed in the translation files, will send the files over again just in case.

Reply:
Looks good to me; thanks!